firstvoices/backend/management/commands/convert_mp3_to_audio.py

```diff
@@ -0,0 +1,84 @@
+import logging
+
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from backend.models import Site
+from backend.models.media import Audio, File, Image, Video
+
+
+class Command(BaseCommand):
+    help = "Convert media models with mp3 files to audio models."
+
+    # add arguments to convert media models for a specific site
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--site",
+            dest="site_slug",
+            help="Site slug of the site to convert media models for (optional)",
+            default=None,
+        )
+
+    @staticmethod
+    def convert_to_audio_model(model):
```

Comment:
A possible issue with this conversion is the original ID of the media model is not preserved when converting to an audio model, but all other data is. If this will be a problem I believe I can save the ID and change it after the deleting on the original model

Reply:
We should check whether any of the affected non-Audio mp3s are related to anything else. If they have no relationships, it's fine to reset the IDs. If they are linked, we need to think a little more about how to preserve those links. I am guessing there are only going to be a couple of cases and we can problem re-link them by hand

Reply:
After doing some research on preprod data there are 39 mp3 images and 10 mp3 videos with related content (characters, dictionary entries, songs and stories). I will look into a solution that keeps the original ids, or at least can preserve the relationship between related media

Reply:
Annoyingly, I suppose we should also check those 39+10 items to see whether the same mp3 was already also added as audio